### PR TITLE
cmd: beacon node API timeouts

### PR DESCRIPTION
**Pull request by gsora**
Add two CLI flags for `run`:
 - `--beacon-node-timeout`, which allows users to specify a custom HTTP timeout used for beacon node API calls
 - `--beacon-node-submit-timeout`, that allows users to specify a custom HTTP timeout used for submission beacon node API calls, like block and attestation submission.

Both flag default to the previous 2 seconds timeout.

category: feature
ticket: none

### CODE REVIEW

File: cmd/run.go
Comment (OisinKyne) [inline cmd/run.go:78]:
```suggestion
	cmd.Flags().DurationVar(&config.BeaconNodeTimeout, "beacon-node-timeout", eth2ClientTimeout, "Timeout for the HTTP requests Charon makes to the configured beacon nodes.")
	cmd.Flags().DurationVar(&config.BeaconNodeSubmitTimeout, "beacon-node-submit-timeout", eth2ClientTimeout, "Timeout for the submission-related HTTP requests Charon makes to the configured beacon nodes.")
```

**Review by OisinKyne — APPROVED** [review]
Not completely sure of app/app.go, but LGTM at least.

**Comment by gsora** [comment]
I'll wait for protocol team's review before merging.

Thanks for your help on the CLI arguments copy!

**Review by pinebit — APPROVED** [review]

**Pull request merged**
